automation/terraform/modules/kubernetes/testnet/variables.tf

```diff
@@ -46,6 +46,16 @@ variable "coda_image" {
   default = "codaprotocol/coda-daemon:0.0.13-beta-master-99d1e1f"
 }
 
+variable "use_custom_entrypoint" {
+  type    = bool
+  default = false
+}
+
+variable "custom_entrypoint" {
```

Comment:
It might make more sense to only have this `custom_entrypoint` variable, and default that to `null`. Then you only need to have one input for setting the entrypoint, and not passing it means you don't want a custom one.

Reply:
I'm not sure there is a null in helm.  in helm/seed-node/values.yaml there's this pair of values:

```  enableArchive: false
  archiveAddress: archive-1:3086```

so I just figured i'd do it the same way.  what do you think?

Reply:
@lk86 what do you think about this?

Reply:
yeah using two variables, a bool to enable the functionality and a string to pass in a value is fairly standard in helm. On the terraform side I see the argument for doing something smarter based on a null value or empty string, but mimicking the helm internals seems equally reasonable.